src/net.cpp

```diff
@@ -2581,30 +2581,17 @@ void CNode::AskFor(const CInv& inv)
     mapAskFor.insert(std::make_pair(nRequestTime, inv));
 }
 
-CDataStream CConnman::BeginMessage(CNode* pnode, int nVersion, int flags, const std::string& sCommand)
+void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg msg)
```

Comment:
Nothing secret is being done. This would fail, for example:
```c++
CSerializedNetMsg x = ...;
pconman->PushMessage(pnode, x);
```
as the binding of the actual parameter x would need a copy constructor to assign to the formal parameter msg.